GNUmakefile

```diff
@@ -43,6 +43,11 @@ test:
 integration:
 	go clean -testcache && go test --tags=integration ./...
 
+report:
+	go get github.com/vakenbolt/go-test-report && go install github.com/vakenbolt/go-test-report/
+	go clean -testcache && go test -json --tags=integration ./... | go-test-report -o  test-report.html -t "Go SDK Tests report"
+	go mod tidy
```

Comment:
I don't think we want to run the go get an tidy here.
If I understand correctly we want to run just "go clean -testcache && go test -json --tags=integration ./... | go-test-report -o test-report.html -t "Go SDK Tests report"" when executing make report.
And the go-test-report package should be setup in go tools or directly in go mod file

Reply:
I cannot put it i the go mod: it created an error because it is not called anywhere in the code. Do you have an idea how I could do it?

Reply:
Try adding to install-tools command it should work